tests/sentry/api/endpoints/test_organization_config_integrations.py

```diff
@@ -27,13 +27,13 @@ def test_provider_key(self):
         assert response.data["providers"][0]["name"] == "Example Server"
 
     def test_feature_flag_integration(self):
-        features.add("organizations:integrations-feature_flag_integration", OrganizationFeature)
+        features.add("organizations:integrations-feature-flag-integration", OrganizationFeature)
```

Comment:
@MeredithAnya I think we need to update https://github.com/getsentry/sentry/blob/7df6bbf4dffb707e5bfccb0cffca4b42b058d97e/tests/acceptance/test_organization_integration_detail_view.py#L20
for consistency